core/domain/role_domain.py

```diff
@@ -0,0 +1,76 @@
+# coding: utf-8
```

Comment:
This file would be better called role_services.py. *_domain.py files are for domain objects.